docs/api.md

```diff
@@ -7,6 +7,8 @@
 * [APIs](#api)
     * [Add Comment](#add-comment)
     * [Get Comments](#get-comments)
+    * [Get Comment Rating] (#get-rating)
```

Comment:
There shouldn't be a space between the name and href, but it seems to work anyways.